src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -132,7 +128,11 @@ public void exec(final URI home) throws IOException {
      * BkBasic produces headers with addresses without slashes.
      *
      * @throws IOException If some problem inside
+     * @todo #519:30min In org.takes.rq.RqHeaders.Base#map is skipping the
```

Comment:
@carlosmiranda I think, I introduce this test failure with my code. I am not sure how i introduce it.
the header is missing `GET /path/c HTTP/1.1`  in the iterator when we run the `addressesInHeadersAddedWithoutSlashes` all other cases it is perfect. 

Any way skipping one item in the iterator here (https://github.com/yegor256/takes/blob/master/src/main/java/org/takes/rq/RqHeaders.java#L142) always and without any comment or instruction why seems a bug  to me.